manifest.json

```diff
@@ -1,24 +1,44 @@
 {
   "manifest_version": 3,
   "name": "Hide Amazon Cart",
-  "version": "0.0.4",
+  "version": "0.0.6",
   "description": "Hide the Amazon cart sidebar while browsing on Amazon.",
-  "permissions": ["scripting"],
+  "permissions": ["scripting", "tabs"],
   "icons": {
     "16": "images/cart-16.png",
     "32": "images/cart-32.png",
     "48": "images/cart-48.png",
     "128": "images/cart-128.png"
   },
   "background": {
-    "service_worker": "scripts/background.js"
+    "service_worker": "scripts/background.js",
+    "scripts": ["scripts/background.js"]
   },
   "web_accessible_resources": [
     {
       "resources": ["css/*.css"],
-      "extension_ids": ["jijclebhnnihgcgjabjgcdbapeccoodc"]
+      "matches": [
```

Comment:
Adding this solely to get rid of the extension id field and Chrome requires two keys in this object.